task1.js

```diff
@@ -0,0 +1,47 @@
+//Task 1: Fix the Lost Context
+
+
+const editor = {
+  title: 'my first blog',
+  getUpperTitle() {
+    return this.title.toUpperCase();
+  }
+};
+
+const getTitle = editor.getUpperTitle.bind(editor);
+
+console.log(getTitle());  
+
+//Task 2: Arrow or Regular?
+
+const formHandler = {
+  value: 'initial',
+  onChange(newValue) {
+    this.value = newValue;
+  }
+};
+
+function simulateInputChange(callback) {
+  callback('updated');
+}
+
+simulateInputChange(formHandler.onChange.bind(formHandler));
```

Comment:
Please use arrow function as another solution without editing the original object, also, please mention why does both solutions work